### ISSUE TITLE
Entered 5 keyword values but only first three are highlighted

### ISSUE THREAD

**Issue by dakavisa** [samtecspg/articulate#877]
<img width="413" alt="Screen Shot 2019-08-20 at 8 21 30 AM" src="https://user-images.githubusercontent.com/53984674/63355580-94d88400-c323-11e9-8f07-0d716cae4beb.png">

And this is the modifieres screen:

<img width="336" alt="Screen Shot 2019-08-20 at 8 22 27 AM" src="https://user-images.githubusercontent.com/53984674/63355689-c7827c80-c323-11e9-8f5d-efa18eba2292.png">

**Comment by wrathagom** [comment]
any errors on the console?

**Comment by dakavisa** [comment]
No, I checked with @dcalvom and I didn't save before moving from values to modifiers, I'm relabelling this as enhancement to maybe sync at UI level both lists and then save everything when the user wants to.

**Comment by dakavisa** [comment]
> any errors on the console?
No, I checked with @dcalvom and I didn't save before moving from values to modifiers, I'm relabelling this as enhancement to maybe sync at UI level both lists and then save everything when the user wants to.
